app/webpacker/src/stylesheets/_tariff-inset.scss

```diff
@@ -0,0 +1,14 @@
+$tariff-inset-background-color-meursing:  #e9f4f9;
+$tariff-inset-border-left-color-meursing: #098dc1;
+
+$tariff-inset-background-color-information:  #f0f4f5;
```

Comment:
should we move these colour definitions to `stylesheets/_colors.scss`?

Reply:
I think you're right. I'll put those in _colors.scss